core/src/main/java/com/graphhopper/util/Constants.java

```diff
@@ -67,7 +67,7 @@
     private static final int JVM_MINOR_VERSION;
 
     public static final int VERSION_NODE = 5;
-    public static final int VERSION_EDGE = 19;
+    public static final int VERSION_EDGE = 20;
```

Comment:
Is there anything else we need to take care of since the `country` EncodedValue now changed it's meaning? Or do we even need to called it something else?

Reply:
Do we even need to do this? Changing the enum constants should yield a different hash of the EV.

Reply:
https://github.com/graphhopper/graphhopper/blob/86d23a3131fb7f65706f2a52ff23bfe250d27c48/core/src/main/java/com/graphhopper/routing/ev/EnumEncodedValue.java#L60-L63

Reply:
Ok, I'll revert this then. This way no new import is required in case the country encoded value was not used at all.